services/static-webserver/client/source/class/osparc/product/quickStart/s4llite/Dashboard.js

```diff
@@ -40,7 +40,7 @@ qx.Class.define("osparc.product.quickStart.s4llite.Dashboard", {
       this._add(dashboardProjects);
 
       const newProjectText = this.tr("\
-        1) Start <i>S4L<sup>lite</sup></i>: Click the <b>+ Start <i>S4L<sup>lite</sup></i></b> button to create a new project. This will start the user interface of <i>S4L<sup>lite</sup></i>.\
+        1) Start Sim4Life.lite: Click the <b>+ Start Sim4Life.lite</b> button to create a new project. This will start the user interface of Sim4Life.lite.\
```

Comment:
same here (an in the other strincts), can't you use e.g. the `displayName` and inject the names in the string?
```javascript
const displayName = "Sim4Life.lite"; // Get this from the corresponding field in the response to `static-frontend-data.json`
const newProjectText = this.tr(`
    1) Start ${displayName}: Click the <b>+ Start ${displayName}</b> button to create a new project. This will start the user interface of ${displayName}.
`);
```